### DISCUSSION TITLE
Sorting table with non-english characters

### DISCUSSION THREAD

**Discussion by hlabory** [quasarframework/quasar#14023] [General - Components / Directives / etc] [answered]
Hello, I am having trouble getting a custom sorting going on the sortable tables.
I set the local to Spanish, and loaded my data, but the sorting doesn’t function properly.

Instead of sorting the non-english characters in place, it pushes them to the end of the list.

I tried to use custom-sort hoping to change anything without luck.
Any help appreciated,
Thanks in advance!

**Comment by MilosPaunovic** [comment]
Please provide a Codepen for someone to be able to take a deeper dive and provide a better support.

**Comment by hlabory** [comment, marked as answer]
Hi:
If field is country for example, add .sort in columns definition:

country.sort = (a, b) => a.localeCompare(b, **'es'**, { sensitivity: '**base**' })
